toptal/granite
# [DEVX-524] Replace obsolete ruby2_keywords def with a gem

Replace an obsolete `Ruby3Compatibility` module with a more reliable gem. We need it to be able to move our platform to Ruby 3. Related [PR](https://github.com/toptal/platform/pull/60704/) on Platform's side.

### Review

- n/a Document code according to [Getting Started with Yard](http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md).
- [x] Get approval.

### Pre-merge checklist


### How to test manually
1. With current Platform `master`
2. Change your ruby version to `3.0.4` like: `rbenv local 3.0.4`
3. Run this spec: `ber spec/api/lib/graphql_api/client/mutations/topscreen/applications/stop_screening_mutation_spec.rb:47`
4. It should fail
5. Change granite version to this branch in Gemfile: `gem 'granite', toptal: 'granite', branch: 'devx-524-replace-ruby2-keywords-to-support-ruby3'`
6. Run the same spec
7. It should pass
